### DISCUSSION TITLE
Redux, Axios, React-Form

### DISCUSSION THREAD

**Discussion by RiyaadHossain** [RiyaadHossain/Hoteltastic-Client#70] [Polls]
Who will implement the redux, Axios & react-form functionality?

**Comment by Developer-Rakib** [comment]
Sob 1 jon na kore sby mile jile korle mone hoy valo hobe

**Reply by khanshorif331** [reply]
Yeah you are right.I also agree with you.

**Comment by khanshorif331** [comment]
Yeah,I also agree with Mr.Rakib.We can devide things in between us.It will be much more effective for all of us.At the same time,pressure will be released from an individual person.Happy Coding...

**Reply by RiyaadHossain** [reply]
Great 😃

**Reply by RiyaadHossain** [reply]
Thanks a lot, you've just reduced my pressure 😌

**Reply by khanshorif331** [reply]
My pleasure..Take love......
